config/foreman.hiera/scenario/katello.yaml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Is this the correct way to handle this file? This worked when I tested it so I assume yes :)

Reply:
In itself it's correct. I am slightly tempted to make it a bit more complex and go for `scenario/%{facts.kafo.scenario.id}/family/%{facts.os.family}-%{facts.os.release.major}.yaml` so we don't attempt to do this on EL8. Maybe that's just premature optimization though.